Dockerfile

```diff
@@ -12,25 +14,24 @@ RUN set -x \
 
 # install helm
 RUN set -x \
-    && curl -s https://storage.googleapis.com/kubernetes-helm/helm-v$HELM_VERSION-linux-amd64.tar.gz | tar xzf - linux-amd64/helm \
+    && curl -s https://get.helm.sh/helm-v$HELM_VERSION-linux-amd64.tar.gz | tar xzf - linux-amd64/helm \
     && chmod +x ./linux-amd64/helm \
     && mv ./linux-amd64/helm /bin/helm \
     && rm -rf ./linux-amd64
 
-# install helm appr (registry) plugin
-RUN set -x \
-    && mkdir -p /home/draughtsman/.helm/plugins \
-    && curl -L -s https://github.com/app-registry/appr-helm-plugin/releases/download/v$APPR_PLUGIN_VERSION/helm-registry_linux.tar.gz | tar xvzf - registry \
-    && mv ./registry /home/draughtsman/.helm/plugins/registry \
-    && chown -R draughtsman:draughtsman /home/draughtsman/.helm
+
 
 USER draughtsman
 
 ADD draughtsman /home/draughtsman/
 
-RUN cd /home/draughtsman/.helm/plugins/registry \
-    && ./cnr.sh upgrade-plugin \
-    && helm registry --help > /dev/null
+# setup default catalog repo
+RUN helm repo add default-catalog https://giantswarm.github.io/default-catalog/
+
+RUN helm repo update
```

Comment:
Is it enough to update the repo once during the build?

Reply:
Yes, we need only one update because migration runs only once and we are not going to release helm-2to3-migration again.